manifests/configmap-seed/remove-persistence-pvc.patch.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mtls
+spec:
+  template:
+    spec:
+      containers:
+        - name: mtls
+          volumeMounts:
+            - $patch: delete
+              mountPath: /home/mtls/database
```

Comment:
Why delete the database?

Reply:
I thought the DB was used for storing which GPG keys are trusted?

Reply:
No, the DB is for certificate storage

Reply:
But isn't there trust data stored in there along with the certificates? I'm looking at e.g. [verify()](https://github.com/drGrove/mtls-server/blob/ea3d8f5179d91966ef4c488ccb078ee0f915a86c/mtls_server/cert_processor.py#L97) and [add_user()](https://github.com/drGrove/mtls-server/blob/ea3d8f5179d91966ef4c488ccb078ee0f915a86c/mtls_server/handler.py#L173).

The premise of this PR is that we get easy user management by just being ephemeral. If a user is removed, but their certificates are still in the DB, then that gives them an unintended level of access, right? Or am I confused?

Reply:
GPG signature is what is determined for authentication, if their certificate is left over they will still not be able to access the service as long as their key has been removed. So they will not receive a new certificate

Reply:
Thanks for explaining. Database: kept.